bootcampspringboot/src/test/java/com/max/bootcampspringboot/archunit/ArchitectureTest.java

```diff
@@ -0,0 +1,134 @@
+package com.max.bootcampspringboot.archunit;
+
+
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.lang.ArchRule;
+import jakarta.persistence.Entity;
+import org.junit.jupiter.api.Test;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+import static com.tngtech.archunit.library.Architectures.layeredArchitecture;
+import static com.tngtech.archunit.library.dependencies.SlicesRuleDefinition.slices;
+
+public class ArchitectureTest {
+    @Test
+    public void packageDependencyCheck() {
+        JavaClasses importedClasses = new ClassFileImporter().importPackages("com.max.bootcampspringboot");
+
+        ArchRule rule = classes().that().resideInAPackage("..api..")
```

Comment:
Du kannst die Ordner, also das "..data.." etc. in Konstanten auslagern, das macht das Ganze lesbarer und wiederverwendbarer. :)

Reply:
weiß nicht ob ich das so nice finde - finde so ist das fast besser lesbar, weil die ja nicht so komplex sind und da straight api steht

Reply:
Du hast ja dann aber eine Konstante die straight API heißt. 
Also private static Final String API = "..api..";